spaces/lex/index.json

```
@@ -0,0 +1,13 @@
+{
+  "key": "$LEX",
```

The key will be the url and we want it all in lowercase and without special characters. Please modify the value of the key by something like `lex`. Thank you.
```suggestion
  "key": "lex",
```